### PR TITLE
modify dockerID to containerID

### PR DESCRIPTION
**Pull request by timyinshi**
*What type of PR is this?**
/kind flake
/sig node

**What this PR does / why we need it**:
modify dockerID to containerID
this change further enforces the usage of container instead of docker in variable names.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:
```docs
NONE
```

### CODE REVIEW

**Review by mattjmcnaughton — APPROVED** [review]
/lgtm

I agree this change is more readable. Do you mind either updating your commit message or PR to include why this change is helpful (i.e. `This change further enforces the usage of container instead of docker in variable names.`).

**Comment by mattjmcnaughton** [comment]
@timyinshi I believe that we need final sign-off from @tallclair before this diff gets merged. Given its KubeCon week and 1.17 release, it might take a couple of extra days :)

**Comment by timyinshi** [comment]
@mattjmcnaughton  that is right.

**Pull request merged**
